src/main/kotlin/me/owdding/catharsis/features/blocks/BlockReplacements.kt

```diff
@@ -31,12 +33,12 @@ import kotlin.time.Duration.Companion.minutes
 import kotlin.time.toJavaDuration
 
 @Module
-object BlockReplacements : PreparingModelLoadingPlugin<Map<Block, LayeredBlockReplacements>> {
+object BlockReplacements : PreparingModelLoadingPlugin<Map<Block, LayeredBlockReplacements>>, CatharsisLogger by Catharsis.featureLogger() {
```

Comment:
the logger shouldnt be exposed like this each plugin should use its own to keep it organized

Reply:
Every plugin uses it's own?